nio/responses.py

```diff
@@ -1736,8 +1736,8 @@ def from_dict(
         )
 
         devices = DeviceList(
-            parsed_dict["device_lists"]["changed"],
-            parsed_dict["device_lists"]["left"],
+            parsed_dict.get("device_lists", {}).get("changed", []),
```

Comment:
I don't think just changing this is enough since they are marked as required over [here](https://github.com/poljar/matrix-nio/blob/master/nio/schemas.py#L463) as well.

Reply:
Good point, I have updated the schema.